### PR TITLE
API documentation

### PR DESCRIPTION
**Pull request by winniederidder**
See https://github.com/dodona-edu/dodona/pull/2158

### CODE REVIEW

File: en/guides/creating-an-api-token/README.md [hunk 1]
```diff
@@ -8,23 +8,17 @@ By creating an API token, you can authorize external applications, such as the [
 
 ## 1. Go to your profile page
 
-First, go to your profile page by clicking the link in the dropdown menu.
+First, go to your profile page by clicking the link in the dropdown menu. Alternatively, you can enter the /profile URL in your searchbar.
```
Comment (rien) [inline en/guides/creating-an-api-token/README.md:11]:
```suggestion
First, go to your profile page by clicking the link in the dropdown menu. Alternatively, you can browse to [dodona.ugent.be/profile](https://dodona.ugent.be/profile).
```

File: nl/guides/creating-an-api-token/README.md [hunk 2]
```diff
@@ -8,23 +8,17 @@ Door een API token aan te maken, kan je andere externe applicaties tot jouw Dodo
 
 ## 1. Ga naar je profielpagina
 
-Ga eerst naar je profielpagina. Dit doe je door op "Mijn profiel" te klikken in het menu onder je naam, rechtsboven op de Dodona website.
+Ga eerst naar je profielpagina. Dit doe je door op "Mijn profiel" te klikken in het menu onder je naam, rechtsboven op de Dodona website. Je kan ook de /profile URL invoeren in je zoekbalk om meteen op deze pagina te komen.
```
Comment (rien) [inline nl/guides/creating-an-api-token/README.md:11]:
```suggestion
Ga eerst naar je profielpagina. Dit doe je door op "Mijn profiel" te klikken in het menu onder je naam, rechtsboven op de Dodona website. Je kan ook surfen naar [dodona.ugent.be/profile](https://dodona.ugent.be/profile).
```

File: en/guides/creating-an-api-token/README.md [hunk 3]
```diff
@@ -8,23 +8,17 @@ By creating an API token, you can authorize external applications, such as the [
...
+## 2. Create a token
 
-At your profile page, click the yellow `EDIT`-button at the bottom.
-
-![Edit your profile](./edit.png)
-
-## 3. Create a token
-
-Scroll to the bottom of the page and generate a new token by clicking the `+`-button. You can choose the description yourself.
+Scroll to the middle of the page and generate a new token by clicking the `+`-button. Fill in a description of your choice. This allows you to identify the token later on, as it tells you which program uses that specific token.
```
Comment (rien) [inline en/guides/creating-an-api-token/README.md:17]:
```suggestion
Scroll to the middle of the page and generate a new token by clicking the `+`-button. Fill in a description of your choice (ex. _pycharm_). This allows you to identify the token later on, as it tells you which program uses that specific token.
```

**Comment by bmesuere** [comment]
This should be merged when the next release is deployed. Converting this to draft to prevent accidental merge.

**Review by bmesuere — APPROVED** [review]

**Pull request merged**